gst/nnstreamer/tensor_filter/tensor_filter_common.h

```diff
@@ -83,6 +83,23 @@ gst_tensor_filter_common_init_property (GstTensorFilterPrivate * priv);
 extern void
 gst_tensor_filter_common_free_property (GstTensorFilterPrivate * priv);
 
+/**
+ * @brief Get available framework from given file when user selects auto option
+ * @param[in] model_files the prediction model paths
+ * @param[in] num_models the number of model files
+ * @return Automatically detected framework name (NULL if the framework is not supported or invalid)
```

Comment:
(NULL if it fails to automatically detect)?

Reply:
Thank you. Updated!